packages/account-v2/src/components/Clipboard/Clipboard.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { useCopyToClipboard } from 'usehooks-ts';
+import { LabelPairedCircleCheckCaptionFillIcon, StandaloneCopyRegularIcon } from '@deriv/quill-icons';
+import { Tooltip, useDevice } from '@deriv-com/ui';
+
+type TProps = {
+    infoMessage?: string;
+    popoverAlignment?: 'bottom' | 'left' | 'right' | 'top';
+    successMessage?: string;
+    textCopy: string;
+};
+
+/**
+ * @depricated TODO: remove this when it's available in @deriv-com/ui. This is temporary.
```

Comment:
can we move this to `@deriv-com/ui` ?

Reply:
@shayan-deriv, I'm not sure if it's good enough or not, but I didn't put in too much effort, as it's a temporary component. In fact I got the one from `Cashier` I didn't create it. I just modified it to use tailwind and fixed a few things

Reply:
ok then. we'll add it later